dags/bqetl_artifact_deployment.py

```diff
@@ -70,14 +87,15 @@ def check_for_queued_runs(dag_id: str) -> bool:
     schedule_interval="@daily",
     doc_md=__doc__,
     tags=tags,
+    params=params,
 ) as dag:
     docker_image = "gcr.io/moz-fx-data-airflow-prod-88e0/bigquery-etl:latest"
```

Comment:
Isn't the generated SQL already included in this image?

Reply:
Yes but if we trigger this from jenkins after a schema deployment, the image won't be rebuilt so generation is run in the tasks.  This is to avoid issues like with the fenix views being outdated https://bugzilla.mozilla.org/show_bug.cgi?id=1904009 although we already have a workaround for this specific issue